examples/cn0565/cn0565_prod_tst.py

```diff
@@ -0,0 +1,201 @@
+import sys
+import time
+import cmath
+import pandas as pd
+import time
+import openpyxl
+from datetime import datetime
+import adi
+from pprint import pprint
+
+# Electrode Names Dictionary
+electrode_name = ["R26_C56_C57", # Electrode 0
+                  "R24_C54_C56", # Electrode 1
+                  "R22_C52_C54", # Electrode 2
+                  "R18_C50_C52", # Electrode 3
+                  "R16_C48_C50", # Electrode 4
+                  "R14_C46_C48", # Electrode 5
+                  "R12_C44_C46", # Electrode 6
+                  "R10_C42_C44", # Electrode 7
+                  "R8_C40_C42", # Electrode 8
+                  "R9_C38_C40", # Electrode 9
+                  "R4_C36_C38", # Electrode 10
+                  "R2_C29_C36", # Electrode 11
+                  "R1_C29_C33", # Electrode 12
+                  "R3_C33_C37", # Electrode 13
+                  "R5_C37_C39", # Electrode 14
+                  "R7_C39_C41"] # Electrode 15
+
+
+def main():
+#----------------------------------------------------------------------------------------------------
+# DEVICE SETTINGS
+#----------------------------------------------------------------------------------------------------
+    cn0565 = adi.cn0565(uri="serial:COM5,230400,8n1n")
+    # reset the cross point switch
+
+    cn0565.gpio1_toggle = True
+    cn0565.excitation_amplitude = 100 # Set amplitude
+    cn0565.excitation_frequency = 10000 # Hz # Set frequency between 10kHz to 80kHz
+    cn0565.magnitude_mode = False
+    cn0565.impedance_mode = True
+
+    cn0565.immediate = True
+
+    cn0565.add(0x71)
+    cn0565.add(0x70)
+
+    fplus = 1
+    splus = 4
+    fminus = 4
+    sminus = 1
+
+    cn0565[fplus][0] = True
+    cn0565[splus][1] = True
+    cn0565[sminus][2] = True
+    cn0565[fminus][3] = True
+
+    # Array for Real & Imaginary values per iteration
+    real_impedance = []
+    imag_impedance = []
+
+    # Pair Count
+    pair = 1
+
+    passed = 0
+
+    Electrode_Iter = iter(range(0,16))
+    for neg_e in Electrode_Iter:
+        pos_e = next(Electrode_Iter)   # setting pos_e = Electrode 0 to neg_e
+
+        if pos_e == neg_e: 
+            print("Forcing electrodes are shorted in the given assignments.")
+            return False
```

Comment:
I don't really understand what this code does. It just provides odd and even numbers... The conditional can't be reached. Can you explain the idea here?

Reply:
It is used for labeling, iteration, and positions of the electrode pairs.

Reply:
Conditional is not reached here. We can remove line 76-78.

Reply:
Removed the if pos_e == neg_e conditional block.